rero/rero-ils
# indexing: update circulation policies mapping

* Adds analyzer on name and description fields.
* Fixes type on description field.

How to test:

* `script/setup`
* http://localhost:5000/admin/records/circ_policies
* Find a term and check if the result is correct

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
